frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -37,6 +37,7 @@ function GridCardGraph({
 	setLayout,
 	onDragSelect,
 }: GridCardGraphProps): JSX.Element {
+	const prevChartDataSetRef = React.useRef<ChartData | undefined>(undefined);
```

Comment:
can we solve without useRef ?

Reply:
I am not sure, @palashgdev 
As basic idea about this solution is to get pervious data and get check accordingly. So it will work.